src/Hl7.Fhir.Base/FhirPath/Functions/UtilityOperators.cs

```diff
@@ -10,6 +10,10 @@ internal static class UtilityOperators
     {
         public static IEnumerable<ITypedElement> Extension(this IEnumerable<ITypedElement> focus, string url)
         {
+            var exts = focus.Navigate("extension").ToArray();
```

Comment:
This seems to double the work?

Reply:
I think you meant to return `matches` here, splitting it up was maybe for debug purposes?

Reply:
Indeed. I removed it again.